app/src/pages/admin/serverSettingsPage/serverSettingsTabs/authConfigurationTab/forms/githubAuthForm/githubAuthFormFields/githubAuthFormFields.jsx

```diff
@@ -44,30 +44,36 @@ const messages = defineMessages({
 
 @connect(
   (state) => ({
-    organizations: (formValueSelector(GITHUB_AUTH_FORM)(state, ORGANIZATIONS_KEY) || '').split(','),
+    organizations: formValueSelector(GITHUB_AUTH_FORM)(state, ORGANIZATIONS_KEY),
   }),
   { showModalAction },
 )
 @injectIntl
 export class GithubAuthFormFields extends Component {
   static propTypes = {
     intl: intlShape.isRequired,
-    organizations: PropTypes.array,
+    organizations: PropTypes.string,
     showModalAction: PropTypes.func,
   };
 
   static defaultProps = {
-    organizations: [],
+    organizations: '',
     showModalAction: () => {},
   };
 
   state = {
     isAddOrganizationClicked: false,
   };
 
-  addOrganization = () => {
+  componentDidUpdate(prevProps) {
```

Comment:
For calculating state from props use `getDerivedStateFromProps`

Reply:
Fixed